website/docs/r/grant.html.markdown

```diff
@@ -74,7 +74,7 @@ The following arguments are supported:
 * `user` - (Optional) The name of the user. Conflicts with `role`.
 * `host` - (Optional) The source host of the user. Defaults to "localhost". Conflicts with `role`.
 * `role` - (Optional) The role to grant `privileges` to. Conflicts with `user` and `host`.
-* `database` - (Required) The database to grant privileges on.
+* `database` - (optional) The database to grant privileges on.
```

Comment:
Please use capital with `optional` to be consistent with those above and below.
Also please specify default like on the line below.

Reply:
totally

Reply:
Ok, please specify also
```
Defaults to `*`, which is all databases.
```